src/www/ui/api/Controllers/UploadTreeController.php

```diff
@@ -355,6 +355,7 @@ public function getTreeView($request, $response, $args)
     $query = $request->getQueryParams();
     $agentId = $query['agentId'] ?? null;
     $flatten = $query['flatten'] ?? null;
+
```

Comment:
Please remove irrelevant changes.

Reply:
Hello @GMishx I have never made any change on that part.

Reply:
But the changes to exist in the PR. Please revert them and squash the commit.

Reply:
Thank you @GMishx  I reverted them and squashed the commit into 1.